packages/core/content-type-builder/admin/src/components/FormModal/contentType/createContentTypeSchema.js

```diff
@@ -73,6 +85,17 @@ const createContentTypeSchema = ({
           return !reservedModels.includes(value?.trim()?.toLowerCase());
         },
       })
+      .test({
+        name: 'pluralNameNotAlreadyUsedInCollectionName',
+        message: getTrad('error.contentType.pluralName-equals-collectionName'),
```

Comment:
Do you know what happens if the user is using another language? Does the English get chosen as a default value for the trad?